pallets/capacity/src/lib.rs

```diff
@@ -452,19 +452,19 @@ impl<T: Config> Pallet<T> {
 		unstaker: &T::AccountId,
 		amount: BalanceOf<T>,
 	) -> DispatchResult {
-		let mut staking_account_details =
+		let mut staking_account =
```

Comment:
Refactored `staking_account_details` -> `staking_account` for consistency in the pallet